torch/csrc/jit/ivalue.h

```diff
@@ -256,6 +311,17 @@ struct IValue {
     return "Invalid Tag";
   }
 
+  // for printing
+  std::string format() const {
```

Comment:
We usually call those `toString` (e.g. `torch::jit::Graph::toString`). It's good to have a consistent convention, because it helps with debugging.

Reply:
Since there is a string type toString is already taken, unless the convention of toType is changed.